database/seeds/DatabaseSeeder.php

```diff
@@ -34,7 +34,7 @@ public function run()
             MockDataUsersSeeder::class,
             MockDataSitesSeeder::class,
             MockDataCampaignsSeeder::class,
-            MockDataNetworkHostsSeeder::class,
+//            MockDataNetworkHostsSeeder::class, # not working - @c3zi why?
```

Comment:
Run `composer install`. Seeds and factories are set as a classmap in `composer.json` file.